controls/stig_ocp4.yml

```diff
@@ -914,12 +911,6 @@ controls:
   - audit_rules_privileged_commands_usermod
   - audit_rules_sudoers
   - audit_rules_sudoers_d
-  - audit_rules_unsuccessful_file_modification_creat
-  - audit_rules_unsuccessful_file_modification_ftruncate
-  - audit_rules_unsuccessful_file_modification_open
-  - audit_rules_unsuccessful_file_modification_open_by_handle_at
-  - audit_rules_unsuccessful_file_modification_openat
-  - audit_rules_unsuccessful_file_modification_truncate
   - audit_rules_usergroup_modification_group
```

Comment:
Do we need to remove the below 4 rules? I can see they are duplicated with lines here at https://github.com/ComplianceAsCode/content/blob/7dc7e06c88204afdada5658928f1642151847bfa/control/stig_ocp4.yml#L967-L970.
I might be wrong as I don't know the background of this PR.

```
  - audit_rules_usergroup_modification_group
  - audit_rules_usergroup_modification_gshadow
  - audit_rules_usergroup_modification_passwd
  - audit_rules_usergroup_modification_shadow
```

Reply:
No, there is no need to remove these rules.
They are part of both STIG controls:
- https://stigaview.com/products/ocp/v1r1/CNTR-OS-000930/
- https://stigaview.com/products/ocp/v1r1/CNTR-OS-000950/

And selecting the same rule in multiple controls is fine. They are added only once to the profile.